simtools/tests/test_simtel_histograms.py

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/python3
+
+import logging
+
+import simtools.io_handler as io
+from simtools.simtel.simtel_histograms import SimtelHistograms
+
+logger = logging.getLogger()
+logger.setLevel(logging.DEBUG)
+
+
+def test_histograms():
```

Comment:
Can this test check that the program actually reads a "known" histogram and assert that there is data?

Reply:
No, it reads a 'known' histogram with data and test if it is reading and plotting it correctly.

I think that is what we expect from this test unit. 
Testing the validity of the file should be done by the module that creates them.

Reply:
But without an assert or a place for the test to fail it is hard to see failures. My suggestion was to create a small test file with one histogram and that this test actually asserts it has the date we expected to read. At least that is what I do when I test the DB.

Reply:
I agree that in general it's always better to have asserts in our test units, but in this case it's such a simple class, I think it's fine to keep it like this by now.

This class do not 'do anything' with the data, apart from reading it and plotting. So I don't see how anything should break in a way that the output data would be different than the input.
If something is wrong, it's more likely that something will break in the process and we will get a failed test.

Reply:
OK!